Missionframework/scripts/client/misc/vehicle_permissions.sqf

```diff
@@ -1,41 +1,17 @@
-private [ "_doeject" ];
+params ["_unit", "_vehicle"];
 
-_doeject = false;
+private _role = assignedVehicleRole player select 0;
```

Comment:
My assumption is that `assignedVehicleRole` will always contain an element in the context of this script, and so `select 0` is an okay statement without first checking for `nil`

Reply:
"Shouldn't happen"™ ^^
I also think it shouldn't happen, due to the event handlers of getInMan and seatSwitchedMan. A simple
```sqf
private _role = assignedVehicleRole player;
if (_role isEqualTo []) exitWith {true};
_role = _role select 0;
```
wouldn't hurt basically. It just makes really sure, even if it shouldn't be necessary at all. But @veteran29 is assigned, so I'm fine with giving him the "last word" on this topic.

Reply:
`private _role = assignedVehicleRole player param [0, "cargo"];`
In case of empty array `_role` will default to "cargo" and function won't do anything.

Reply:
I had some cases where it returned `Cargo` instead of `cargo`. Done it in a different way, ref: https://github.com/KillahPotatoes/KP-Liberation/commit/ab637f09d8d7de7b833429c1e7e5b5a2dd39c1cd#diff-df745d117198d8466adb8775321b2c8fR5